webpack.config.dev.js

```diff
@@ -12,15 +12,12 @@ const insertHotReload = (host, port, entries) => (
   R.map(R.compose(R.flatten, v => [v].concat(hotEntry(host, port))), entries)
 );
 
-const devConfig = {
+Object.assign(config, {
   context: __dirname,
   output: {
     path: path.resolve('./static/bundles/'),
     filename: "[name].js"
   },
-  module: sharedConfig.module,
-  sassLoader: sharedConfig.sassLoader,
```

Comment:
I can't remember what I changed that made me not need to add it in the shared config, I think it was that a newer version of the sass loader deprecated that way of doing things or something.